lstchain/calib/camera/calibration_calculator.py

```diff
@@ -77,7 +77,8 @@ class CalibrationCalculator(Component):
 
     hg_lg_ratio = traits.Float(
         1.,
-        help='HG/LG ratio applied if use_scaled_low_gain is True. In case of calibrated data the ratio should be 1.'
+        help='HG/LG ratio applied if use_scaled_low_gain is True. The ratio is ~1 for calibrated data, ~17.4 for uncalibrated data.' 
+
```

Comment:
Can we maybe change this into two options?

The `hg_lg_ratio` should be fixed (at least for LST), to 17.4.

It only needs to be applied for Cat-A. 

So maybe it's simpler to leave that as 17.4 and have a switch `calibrated_input` or simular, that is set to `False` by the cat A calibration script but to `True` by the cat B script?

Or even automatically determined by the input / if an existing calibration file has been given?

Reply:
We never used this option for the Cat-A, it is instead useful for Cat-B calibration because pixels with low HV (due to high NSB) can not be correctly calibrated with LG. For the status of the code/data now, still in test phase, it seems the me that such a trailet is enough to deal with all cases (by experts). I do not see the necessity of an automatisation at this level (but a test can be eventually added at a higher level).